docs/virtual_machines/disks_and_volumes.md

```diff
@@ -944,7 +944,7 @@ fewer IOThreads than CPU, each IOThread will be pinned to a set of CPUs.
 #### IOThreads with QEMU Emulator thread and Dedicated (pinned) CPUs
 
 To further improve the vCPUs latency, KubeVirt can allocate an
-additional dedicated physical CPU[1](creation/dedicated-cpu.md), exclusively for the emulator thread, to which it will
+additional dedicated physical CPU[1](virtual_machines/virtual_hardware/#cpu), exclusively for the emulator thread, to which it will
```

Comment:
The link gives me 404

https://deploy-preview-389--kubevirt-user-guide.netlify.app/virtual_machines/disks_and_volumes/virtual_machines/virtual_hardware/#cpu

Reply:
Investigating that

Reply:
OK, I see what's happenin' here ... We have half of an internal link and half an external link.  The two halves are creating the failure.

We have 19 cases of this in the user-guide.  I'm going to update them all.

Stand by.

Reply:
Link is fixed per local test and Netlify